doc/api/deprecations.md

```diff
@@ -582,6 +582,16 @@ deprecated.
 *Note*: `OutgoingMessage.prototype._renderHeaders` was never documented as
 an officially supported API.
 
+<a id="DEP0068"></a>
+### DEP0068: vm.runInDebugContext(string)
+
+Type: Runtime
+
+The DebugContext is being removed from V8 5.8+ along with the legacy debugger
+interface.
```

Comment:
This is actually not accurate. The Debug context is slated to go away at the end of the year which would probably be either M63 or M64. It might be better to state the Node version (10.0) that this is expected to land in.

The legacy debugger interface is gone in V8 5.8.